build/aznb.ps1

```diff
@@ -0,0 +1,33 @@
+# This script is useful for testing integration with Azure Notebooks.
+# It will replace the language server the Azure Notebooks setup script has
+# downloaded and extracted with your local version of the language server.
+
+$ErrorActionPreference = 'Stop'
+
+& "$PSScriptRoot/set-env.ps1"
+
+# Based on your personal setup, you may need to change these two lines
+$DotNetRuntimeID = "win10-x64"
+$TargetDir = Join-Path $PSScriptRoot "../../AzureNotebooks/src/service/aznb/NotebookLanguageServer/dist/qsharp"
```

Comment:
I'm not sure if this script belongs in this repository, but I also didn't want to hoard it all to myself. What do y'all think?